portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
```

Comment:
Please update the comment of the function

Reply:
Thank you. I changed the description.

Reply:
I think you forgot to update the comment